docs/configuration.rst

```diff
@@ -526,19 +526,20 @@ The KSQL CLI image will execute the ``ksql`` command and try to connect to the s
 
   docker run -it -rm confluentinc/cp-ksql-cli:4.1.2
 
-Note: Since the CLI is interactive, we cannot log to stdout as is customary for Docker containers. The KSQL CLI logs can instead be found at ``/var/logs/ksql-cli/``.
+Note: Because the CLI is interactive, the container cannot log to STDOUT as is customary for Docker containers. The KSQL CLI logs can instead be found at /var/logs/ksql-cli/ inside the container.
```

Comment:
I'd retain the backticks around the path, i.e. it should be ``/var/logs/ksql-cli/``